realis.go

```diff
@@ -548,6 +549,33 @@ func (r *realisClient) GetJobUpdateSummaries(jobUpdateQuery *aurora.JobUpdateQue
 	return resp, nil
 }
 
+func (r *realisClient)	GetJobs(role string) (*aurora.Response, *aurora.GetJobsResult_, error) {
+	var resp *aurora.Response
+	var result *aurora.GetJobsResult_
+	var clientErr error
+
+	retryErr := ExponentialBackoff(*r.config.backoff, func() (bool, error) {
+		resp, clientErr = r.thriftCallHelper(func() (*aurora.Response, error) {
+			return r.readonlyClient.GetJobs(role)
+		})
+
+		if clientErr != nil {
+			return false, clientErr
+		}
+		return true, nil
+	})
+
+	if resp != nil && resp.GetResult_() != nil {
```

Comment:
resp != nil is already checked for inside of thriftCallHelper so we can just get away with checking if resp.GetResult_() is nil here.

Reply:
e2e is updated.. good reminder.
we should add other missing apis as well..